contrib/grv_proxy_model/proxy_model.py

```diff
@@ -149,17 +157,19 @@ def can_start(self, params):
         return params.num_started + params.count <= self.limit
 
     def update_budget(self, params):
-        #if params.num_started > 0:
-            #print('Start update budget: time=%f, limit=%f, locked_until=%f, num_started=%d, priority=%s, min_priority=%s, last_batch=%d' % (params.time, self.limit, self.locked_until, params.num_started, self.priority, params.min_priority, params.last_batch))
+        # if params.num_started > 0:
+        # print('Start update budget: time=%f, limit=%f, locked_until=%f, num_started=%d, priority=%s, min_priority=%s, last_batch=%d' % (params.time, self.limit, self.locked_until, params.num_started, self.priority, params.min_priority, params.last_batch))
 
         if params.num_started > self.limit:
-            self.locked_until = min(params.time + 2.0, max(params.time, self.locked_until) + penalty/self.rate)
+            penalty = 1
```

Comment:
Was this line meant to be added?

Reply:
Yes, I think penalty is undefined. I set it on 1 because of my findings [here](https://github.com/apple/foundationdb/blob/main/fdbrpc/include/fdbrpc/LoadBalance.actor.h#L506-L507). I think 1 is the default value because of this.

Reply:
I see, good catch.